maps_dashboards/opensearch_dashboards.json

```diff
@@ -4,6 +4,6 @@
   "opensearchDashboardsVersion": "3.0.0",
   "server": true,
   "ui": true,
-  "requiredPlugins": ["navigation", "opensearchDashboardsReact", "savedObjects"],
+  "requiredPlugins": ["navigation", "opensearchDashboardsReact", "savedObjects", "data"],
```

Comment:
nit: Not for this PR, but if you only need to import stuff from a plugin but dont need its start and setup services, you can use the `requiredBundles` prop instead.

Reply:
Got it, thanks! I'd like to keep it for now because we might have other use case for data plugin, but thanks let me know for this option!